### ISSUE TITLE
Calibration

### ISSUE THREAD

**Issue by Niall01** [georgejecook/BostoTabletDriverMac#1]
When i change the offset for my pen it only works for that particular , so in general it is not calibrating to the 4 corners of the screen and only from the centre of the screen, so if i move closer to the edge of the screen the further away the cursor gets. i don't know if i'm doing something wrong or if i have my settings wrong or my wacom driver is messing with this one.. seems to be the only issue so far

**Comment by georgejecook** [comment]
I've also noticed a little bit of offset. I had my screen in 1920x1080 (virtualized high def); which I thought was the problem.

I'll try and investigate it this weekend.

**Comment by Niall01** [comment]
I got it working last night perfectly , but a part of the screen was unusable for the Pen, But once i restarted my computer this morning everything i did seemed worthless and am having the same problem again when the cursor moves away from the pen the further away from the centre of the screen you go, I am Going to assume that it is just the display settings fighting with each other, is there any particular setting i need to have my Mac's display settings alongside the Bosto's display settings.

**Comment by georgejecook** [comment]
This is a bug. Can you please tell me your resolution. I use 1920x1080, and i notice the problem a bit by the edges - not really very much, but a little bit...

What device do you have? 19ma/19mb?
What os are you running?
What resolution are you running?

**Comment by Niall01** [comment]
At the moment My display settings are,
iMac Resolution : 1920x1080
Bosto Resolution : 1440x900
I was running Snow leopard 10.6.8 , but the driver needed 10.8 or higher, so i have upgraded my operating system as of last night to Mountain Lion OS X 10.8.3.

And i have a Bosto Kingtee 19mb

**Comment by Niall01** [comment]
http://www.youtube.com/watch?v=HK2gJnGeNFI
heres a video explaining my problem .. hope the video reference helps.

**Comment by georgejecook** [comment]
Transmisión. But the video is prívate. Algo what versión of OS X is  it? Which tahblet model. Thanks

**Comment by Niall01** [comment]
I am running OSX 10.8.3 Mountain Lion.
Tablet model, 19MB

**Comment by thadrazdow** [comment]
I'm having this issue as well. mid 2009 macbook pro, osx 10.8.3. effect is exaggerated when in mirroring mode, but happens either way. I have tried disabling mous acceleration in lion with no effect. any updates on this?

**Comment by Niall01** [comment]
Hi, i'm not exactly very technical minded here, but i had the same problem and as a short term solution i have found if your offset is still off by a lot then a suggestion would be what i did, now this is just a short term solution for me you don't have to take this into consideration because im sure it's probably more hassle in the long run than is needed, i'm just using it this way until George has had enough time to develop his driver more,

The way i have got my offset as close as i possibly can is i have stopped using the VGA cable to my mac and instead have started using HDMI to mini display port that or DVI to mini display, this way when you are in your display settings in your system preferences it gives you an option for "Overscan" and i have found it works much better with mirrored displays as well with your bosto as the primary monitor, Now if you turn overscan off it will change your display size by a good bit, but in result it will keep your cursor under the pen and it only strays by a few millimetres but nothing to much to complain about in my case. you will lose some drawing estate to work with but it's an ok solution if you have enough time with the tablet to wait for some updates on the driver. This might be an invalid solution in some cases but i though I would throw it in there for whatever little help it may serve.

**Comment by Niall01** [comment]
Also for what i just suggested you need to have the driver installed but the driver menu closed for some reason, i think it's because of the offset problems with the X and Y axis on the driver i'm not sure but that is how i got mine to work to the best of it's capabilities in these early stages of development

**Comment by fredewis** [comment]
im having the same problem I have a macbook pro on osx 10.8.3 
would be great to be able to use this tablet if there is anyway i can help would be great.

**Comment by georgejecook** [comment]
Ok,

what model tablet do you have?

If someone can get a recording on an iphone/camera to properly show me the problem; that will greatly help.

thx

**Comment by fredewis** [comment]
i have the 19mb ......

**Comment by georgejecook** [comment]
Ok, can you please explain what the problem is a bit clearer..

People have very different definitions of "the same"... :)

Also, have you ensured that you've selected your bosto tablet in the "general" tab?

**Comment by fredewis** [comment]
https://docs.google.com/file/d/0B2Pnc6LK4xrdSE9BRU5uX1czNlE/edit?usp=sharing
here is a google frive video.

**Comment by georgejecook** [comment]
Hi Freddee, thanks for the video.

Can you please tell me if your monitor is selected in the general tab, and if the status is "connected"?

You and I have the same model, and the same os, yet my one does not exhibit this behaviour...

I think I will have to make a callibration tool...

I'll try and do it tomorrow if I get a chance (snowed now); but I'll get something done this week.. as drawing will not be difficult for you.. it will be impossible with the pen like that (unless you're drunk ;).

Video helps a ton. I'll ping you when I have an update for you to try. 

This is rather sad becuase

**Comment by fredewis** [comment]
ill try the general tab. i dont think i have turned that on.

**Comment by fredewis** [comment]
ooo do you mean general as in the bosto driver general ...or mac os general setting ?

**Comment by georgejecook** [comment]
I mean the bost driver general tab - so you click on the tablet and select "general" - there you can select which window you wish to use the tablet in, and it says the tablet status.

**Comment by fredewis** [comment]
yes i selected the bosto ...will try again in a bit my house mate has desided to plug her windows pc on it to give it a spin  -_-.

**Comment by fredewis** [comment]
no no difference :(

**Comment by fredewis** [comment]
Hi George you probably busy just wondered if you had any chance on tinkering with the driver and if there is anything I can do for you?

**Comment by bsimser** [comment]
@georgejecook Just wondering if you can build the driver for an older OSX? I'm running 10.7.5 but the driver I just tried needs 10.8. Or is it not compatible? Also it's interesting as I flipped over to OSX from Windows and with no driver loaded, I still get pressure sensitivity inside of SketchBook Pro. The calibration is off by a few inches and I have no way of adjusting it though.

**Comment by fredewis** [comment]
sorry for the bother just wondered if there is any news on future fixes

**Comment by georgejecook** [comment]
Sorry Chaps - I've moved house, and been doing all kinds of stuff : plus family illness - and my baby is due in 2 weeks time, so it's been too hard to get round to getting anything done. 

worse of all the tablet is not with me - it's in another city.. so I can't even test it.. I'll try and get some of this stuff done in a few weeks. I'm afraid I can't promise any better right now.

g.
